code/game/objects/structures/flora.dm

```diff
@@ -720,6 +720,83 @@
 	mood_change = -6
 	timeout = 5 MINUTES
 
+/obj/structure/flora/tree/srm
```

Comment:
this code as a whole is... fine, though there's just enough overlap with the chapel tree that i'm tempted to say to subtype it. i'd suggest moving both trees to the bottom of a file and noting their section with a comment, so that they're not mixed in with generic flora structures

Reply:
It was originally a sub type but nearly every proc is changed so there is not much a point in sub typing it